src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -172,7 +181,7 @@ protected function failedAuthorization()
      */
     public function validated()
     {
-        return $this->getValidatorInstance()->validated();
+        return $this->validator->validated();
```

Comment:
What if you call this method before the validator is set? I think we should think this through more.

Reply:
@driesvints 
Thsnks for checking.
How about adding `ValidatesWhenResolvedTrait::validateResolved()` ?

Reply:
@driesvints 
Thanks for checking.
`ValidatesWhenResolvedTrait::validateResolved()` is called in `FormRequestServiceProvider::boot()`.

So I think that it is normally set.

Reply:
can't this be fixed by adding a simple if statement that can be added to the `getValidatorInstance`?

Reply:
@mohamed-aiman 
I think so too, but getValidatorInstance called twice.
We require single validator instance.